### ISSUE TITLE
Allow setting default workspace option

### ISSUE THREAD

**Issue by ortonomy** [benweet/stackedit#1655]
Please enable setting a default workspace other than GDrive. I want to use github as my primary storage option for markdown, but every time that I close the app and re-open, it automatically opens my GDrive and I have to switch 'workspaces'. I'm not really interested in using GDrive, but I had to default to it, because from a UX point of view, it's annoying to switch EVERY time I start the app.

**Comment by Fantantonio** [comment]
I agree; the problems is the same if you've multiple GDrive workspaces and you want always to open a specific one.

It would be a huge help to have a dedicated option in settings
